backend/typescript/services/implementations/authService.ts

```diff
@@ -177,26 +180,28 @@ class AuthService implements IAuthService {
     }
 
     try {
-      const passwordResetLink = await firebaseAdmin
-        .auth()
-        .generatePasswordResetLink(email);
-      // first-time determines if we are setting a new account password
-      // (account will be verified) or reseting an old account's password
-      const setPasswordLink = passwordResetLink.concat("&first-time=true");
+      const authId = await this.userService.getAuthIdById(user.id);
+
+      const setPasswordLink = `${process.env.CLIENT_URL}/auth/action?mode=verify-user&uid=${authId}`;
 
       const emailBody = `
       Hello,
       <br><br>
       You have been invited to join CCBC as a ${user.roleType.toLowerCase()}. Please use the link 
-      below to set your new password and verify your account. The link expires in 1 hour.
+      below to set your new password and verify your account. 
+      <br> Your unique access code is ${accessCode}.
       <br><br>
-      <a href=${setPasswordLink}>Set password and verify account</a>
+      <a href=${setPasswordLink}>Setup Account</a>
       `;
 
-      this.emailService.sendEmail(email, "CCBC Account Created", emailBody);
+      this.emailService.sendEmail(
+        user.email,
+        "CCBC Account Created",
```

Comment:
Should the subject be: `CCBC Verify Account`?

Reply:
I think Account Created is a better subject imo. The purpose of the email is to invite the person to finish setting up an admin account for themselves. The account is "created" for them by the previous admin. This email notifies them that the account is created. Happy to change it to verify though if you so think that's better

Reply:
I think something like `Verify your CCBC Account` (prob consult designers for exact copy) tells the user that there is action needed on their part.

Reply:
Updated the subject to "Verify your CCBC Account" as requested